### PR TITLE
Add two more examples for applications of DeepXDE

### PR DESCRIPTION
**Pull request by mariusmerkle**
- Kovasznay flow
- Beltrami flow

### CODE REVIEW

**Comment by lululxvi** [comment]
Thank you for the code. It looks good, but only one thing to do: Could you format the code using black https://github.com/psf/black?

**Comment by mariusmerkle** [comment]
Hi Lu, I have formatted both files with the default settings of black, so it should be fine now.

File: examples/Beltrami_flow.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    l2_difference_w_0 = dde.metrics.l2_relative_error(w_exact_0, w_pred_0)
+    l2_difference_p_0 = dde.metrics.l2_relative_error(p_exact_0, p_pred_0)
+    residual_0 = np.mean(np.absolute(f_0))
+
+    l2_difference_u_1 = dde.metrics.l2_relative_error(u_exact_1, u_pred_1)
+    l2_difference_v_1 = dde.metrics.l2_relative_error(v_exact_1, v_pred_1)
+    l2_difference_w_1 = dde.metrics.l2_relative_error(w_exact_1, w_pred_1)
+    l2_difference_p_1 = dde.metrics.l2_relative_error(p_exact_1, p_pred_1)
+    residual_1 = np.mean(np.absolute(f_1))
+
```
Comment (lululxvi) [inline examples/Beltrami_flow.py:226]: Do you want to print something?

File: examples/Kovasznay_flow.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    def u_func(x):
+        return 1 - np.exp(l * x[:, 0:1]) * np.cos(2 * np.pi * x[:, 1:2])
+
+    def v_func(x):
+        return l / (2 * np.pi) * np.exp(l * x[:, 0:1]) * np.sin(2 * np.pi * x[:, 1:2])
+
+    def p_func(x):
+        return 1 / 2 * (1 - np.exp(2 * l * x[:, 0:1]))
+
+    spatial_domain = dde.geometry.geometry_2d.Rectangle(
```
Comment (lululxvi) [inline examples/Kovasznay_flow.py:50]: `dde.geometry.Rectangle`

File: examples/Kovasznay_flow.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        xmin=[-0.5, -0.5], xmax=[1, 1.5]
+    )
+
+    def boundary_outflow(x, on_boundary):
+        return on_boundary and spatial_domain.on_boundary(x) and np.isclose(x[1], 1.5)
```
Comment (lululxvi) [inline examples/Kovasznay_flow.py:55]: `on_boundary` is `spatial_domain.on_boundary(x)`.
```
        return on_boundary and np.isclose(x[1], 1.5)
```

File: examples/Kovasznay_flow.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    v_exact = v_func(X).reshape(-1)
+    p_exact = p_func(X).reshape(-1)
+
+    f = model.predict(X, operator=pde)
+
+    l2_difference_u = dde.metrics.l2_relative_error(u_exact, u_pred)
+    l2_difference_v = dde.metrics.l2_relative_error(v_exact, v_pred)
+    l2_difference_p = dde.metrics.l2_relative_error(p_exact, p_pred)
+    residual = np.mean(np.absolute(f))
+
```
Comment (lululxvi) [inline examples/Kovasznay_flow.py:101]: Do you want to print something?

File: examples/Beltrami_flow.py (hunk 1 above)
Comment (mariusmerkle) [reply examples/Beltrami_flow.py:226]: Yes, now the accuracy at the t = 0 (satisfaction of initial conditions) and t = 1 are printed. For the accuracy, both the PINN loss (PDE residual) and the relative L2 difference with respect to the analytic solution are printed.

File: examples/Kovasznay_flow.py (hunk 4 above)
Comment (mariusmerkle) [reply examples/Kovasznay_flow.py:101]: Yes, again the PINN loss (PDE residual) and the relative error with respect to the analytic solution are printed.

File: examples/Beltrami_flow.py (hunk 1 above)
Comment (mariusmerkle) [reply examples/Beltrami_flow.py:226]: This are the results from a test _run:_

Accuracy at t = 0:
Mean residual: 0.0088581275
L2 relative error in u: 0.0003874505051137696
L2 relative error in v: 0.0004037125060594724
L2 relative error in w: 0.00039577592611241826

Accuracy at t = 1:
Mean residual: 0.004351791
L2 relative error in u: 0.0020703697359720956
L2 relative error in v: 0.0015207481601198175
L2 relative error in w: 0.001569895783311284

File: examples/Kovasznay_flow.py (hunk 4 above)
Comment (mariusmerkle) [reply examples/Kovasznay_flow.py:101]: These are the results from a test run:

Mean residual: 0.00062245055
L2 relative error in u: 0.00018655356498004745
L2 relative error in v: 0.0008943926276830373
L2 relative error in p: 0.0003445579388838097

**Comment by mariusmerkle** [comment]
Hi Lu,

I have integrated all your suggestions and formatted the files with black again.

**Pull request merged**